Add links to the install- and dev-manuals (#4)
Gamify-IT/.github: profile/README.md

For an in depth explanation of each service go to the **[user manuals](https://gamifyit-docs.readthedocs.io/en/latest/user-manuals/index.html)**

## Installation Manuals

> For informations about the configuration of services go to the **[installation manuals](https://gamifyit-docs.readthedocs.io/en/latest/install-manuals/index.html)**

Reviewer: ```suggestion
To learn how to install everything/ specific services, visit the **[installation manuals](https://gamifyit-docs.readthedocs.io/en/latest/install-manuals/index.html)**.
```

## Developer Manuals

> Documents for Developers with a lot of knowledge about the projects can be found in the **[developer manuals](https://gamifyit-docs.readthedocs.io/en/latest/dev-manuals/index.html)**

Reviewer: ```suggestion
Specific knowledge about services/ the architecture can be found in the **[developer manuals](https://gamifyit-docs.readthedocs.io/en/latest/dev-manuals/index.html)**.
```

## Installation Manuals

> To learn how to install everything, visit the **[installation manuals](https://gamifyit-docs.readthedocs.io/en/latest/install-manuals/index.html)** For a template on how to host checkout the [run config](https://github.com/Gamify-IT/run-config).

Reviewer: ```suggestion
To learn how to install everything, visit the **[installation manuals](https://gamifyit-docs.readthedocs.io/en/latest/install-manuals/index.html)**. For a template on how to host checkout the [run config](https://github.com/Gamify-IT/run-config).
```
